services/compileGRPCStubs/template/downloadStubFiles.ts

```diff
@@ -0,0 +1,85 @@
+require("dotenv").config({path:__dirname+'/../.env'});
+const path = require("path");
+const fs = require("fs");
+var s3 = require("s3-client");
+var rimraf = require("rimraf");
+const globby = require("globby");
+
+
+var client = s3.createClient({
+  maxAsyncS3: 20, // this is the default
+  s3RetryCount: 3, // this is the default
+  s3RetryDelay: 1000, // this is the default
+  multipartUploadThreshold: 20971520, // this is the default (20 MB)
+  multipartUploadSize: 15728640, // this is the default (15 MB)
+  s3Options: {
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+    region: process.env.AWS_REGION,
+  },
+});
+
+var params = {
+  localDir: "./grpc_stubs/",
+  s3Params: {
+    Bucket: process.env.AWS_BUCKET_GRPC_STUBS,
+  },
+};
+
+exports.default = async function (orgId = "6ce80f485dae487688c3a083688819bb", serviceId = "test_freecall") {
```

Comment:
why this orgId and serviceId is hardcoded. could you check this.

Reply:
Hardcoded, for now, the API still gets merged to Ropsten, till then we can only hardcode the values. Parallelly I'll update to dynamic one